### PR TITLE
Updated Charter based on comments in the doc

### PR DESCRIPTION
**Pull request by colinmurphy01**
Changes based on @abrouilette and @kfoley-18F's review of the draft doc.

### CODE REVIEW

File: docs/charter.md [hunk 1]
```diff
@@ -115,9 +131,11 @@ need to ensure that our work is discoverable and documented. To that end;
   where we will collect all related documents
 - All work should be submitted as an issue, either in the Delivery Assurance 
   Github repo or in one of the associated repos
-- On the first day of the iteration, there will be a backlog prioritization
-  session. Alan, along with any team members, will determine what work to do
-  during that iteration and pull it into the to-do column
+- Prior to the next 2-week sprint, 18F Leadership members and the Team Lead will get
+  together to write out [goals](url), project team make-up, and refine the backlog
```
Comment (Sgtpluck) [inline docs/charter.md:135]:
```suggestion
  together to write out [goals](https://docs.google.com/document/d/1u71mGgZmnL9w3w6ylO_n_tKT7RHJ716QmGkgY7BJC2k/edit#heading=h.km5kn21wqypa), project team make-up, and refine the backlog
```

**Review by Sgtpluck — APPROVED** [review]
just a little addition -- i think that's the right url?

File: docs/charter.md (hunk 1 above)
Comment (colinmurphy01) [reply docs/charter.md:135]: Actually I'm looking to point it to a goals.md file in a separate PR the google doc is a drafting space.

Comment (Sgtpluck) [reply docs/charter.md:135]: oh okay! i'll resolve this comment!

**Pull request merged**